borg/lrucache.py

```diff
@@ -1,42 +1,41 @@
-class LRUCache(dict):
```

Comment:
Yes, because all the modifying dict methods are wrong, and it's error-prone to try and override them all.

Another problem is methods which return a dict which copies items.  In the general case you would want to wrap them and return your own class instead.  And since we own the items (we have responsibility for close()), this gets pretty malformed.

It would get even more interesting if a new version of python added a new convenience method :).  It's happened in the past.